lib/mount.js

```diff
@@ -1,22 +1,22 @@
 const { compose, identity, is, when, tap } = require('ramda')
 const { assocWith } = require('@articulate/funky')
 
-const bufferStreams  = require('./bufferStreams')
-const cleanRequest   = require('./cleanRequest')
-const defaultLogger  = require('./logger')
-const error          = require('./error')
-const eventToRequest = require('./eventToRequest')
-const getBody        = require('./getBody')
-const parseCookies   = require('./parseCookies')
-const parseUrl       = require('./parseUrl')
-const wrap           = require('./wrap')
-const writeHTTP      = require('./writeHTTP')
-const writeLambda    = require('./writeLambda')
+const { bufferStreams }  = require('./bufferStreams')
+const { cleanRequest }   = require('./cleanRequest')
+const defaultLogger      = require('./logger').logger
```

Comment:
Any reason not `{ logger: defaultLogger }` for symmetry?

Reply:
Only because that would push the line of `=` even further to the right.  😅 If you think that's silly, I can change it.